### PR TITLE
dropping support for ie8

### PR DESCRIPTION
**Pull request by trevorah**
ie8 is the last supported version of internet explorer for windows xp. It's usage is 1%-4% depending on which site you ask.

At the moment, we teach `$.each` as the way to iterate through an array, and I'd rather have students know the built-in array functions. Dropping ie8 means we can update our material, which is what this pull request does.

Thoughts?

### CODE REVIEW

**Comment by richardwestenra** [comment]
Thanks Andy, this is great! 👌

Thoughts: On the one hand, this is the jQuery tutorial not the JS native Array method tutorial. On the other hand, you're 100% right, we should be teaching `Array.prototype.forEach` instead of `$.each`. While this section was probably originally intended to teach the jQuery method, I'm happy with this change.

I wonder whether we should include a section on advanced native Array/Object methods (e.g. `map` and `reduce`) at the end of tutorial 2, but that's a different topic.

File: js/lesson4/tutorial.md [hunk 1]
```diff
@@ -289,7 +289,7 @@ function retrieveGenres() {
 
 > You can use `<title>` to display a humanly readable format of the Genre
 
-As you can see from the console, the resulting objects are returned inside an Array. We want to iterate over the list using the `$.each( )` function and add each item to the `#genres` list, as a **list item**. As we need to have access to the `key` as well, we can set that as the list item's `id`.
+As you can see from the console, the resulting objects are returned inside an Array. We want to iterate over the list using the `forEach( )` function and add each item to the `#genres` list, as a **list item**. As we need to have access to the `key` as well, we can set that as the list item's `id`.
```
Comment (richardwestenra) [inline js/lesson4/tutorial.md]: We should probably clarify here that we mean the native method. How about we change the following
> using the `forEach( )` function
to something like
> using the native Array `forEach( )` method

**Comment by trevorah** [comment]
I totally agree with splitting things out later on, especially for introducing terms like "method" rather than just "function". It was a bit tricky to talk about forEach without mentioning methods.

 Like you said though, we can always introduce that later.

**Comment by trevorah** [comment]
No problem!

**Pull request merged**